src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/GraphQl/PatientTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Hl7.Fhir.Model;
+using HotChocolate;
+using HotChocolate.Execution;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Api.Features.GraphQl.DataLoader;
+using Microsoft.Health.Fhir.Shared.Api.Features.GraphQl;
+using Snapshooter.Xunit;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Api.UnitTests.Features.GraphQl
+{
+    public class PatientTests
+    {
+        [Fact]
+        [System.Obsolete]
+        public async System.Threading.Tasks.Task<Task> GetPatientsAsync()
+        {
+            // arrange
+            IRequestExecutor executor = (IRequestExecutor)new ServiceCollection()
+                .AddRouting()
+
+                // Adding the GraphQL server core service
+                .AddGraphQLServer()
+
+                // Adding our scheme
+                .AddDocumentFromFile("./patient.graphql")
```

Comment:
>patient [](http://example.com/codeflow?start=40&length=7)

can this files live in Core\Data\GraphQL folder?
I would also advice to have IGraphQL